My first pull request for Hello Microverse Project!

paulinagonzalezc opened the pull request:
In this pull request, I included some example files such as: index.html with a title and styles.css with a basic style for the title. I also set up a linters file to analyze my source code.

paulinagonzalezc committed:
Added github linters file

paulinagonzalezc committed:
Added linters file

paulinagonzalezc committed:
Added .gitignore file

paulinagonzalezc committed:
Edited README File

paulinagonzalezc committed:
Edited .github file

paulinagonzalezc committed:
Added index.html file

paulinagonzalezc committed:
Edited linters.yml file

paulinagonzalezc committed:
Added H1 for html file

paulinagonzalezc committed:
AddedCSS Styles file

paulinagonzalezc committed:
Changed H1 color

paulinagonzalezc committed:
Added test.md file

mwanawabangona commented on README.md:
```diff
@@ -1 +1,15 @@
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
```
- [ ] `required`
- [x] Please consider writting a more descriptive `readme` including social handle like `twitter` & `linkedin` so that recruiters can easily navigate you. please follow this [template](https://github.com/microverseinc/readme-template)

mwanawabangona commented on test.md:
```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```
- [ ]  [required]

- [ ] Please add the test.md file to the gitignore file because The content of this file is not relevant to this project

mwanawabangona reviewed (changes requested):
Hi @paulinagonzalezc ,

Good job so far!
![](https://camo.githubusercontent.com/38ce8c8b213b08e2d9bb85a0b79702ad512f3910f803ec3d7917b4e0110c9044/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966)
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️

- [x] `required`
- [x] you may consider when writing a Pull request always add a short summary of changes introduced in the pull request. please implement this chnage.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Going through the requirements once you are done with your project, step by step ensure you have not missed any. 🥂  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

paulinagonzalezc committed:
Edited the Read Me file to add a more extense description including social handle

paulinagonzalezc committed:
Moved test.md file to gitignore

paulinagonzalezc commented:
Hey @mwanawabangona, I made the changes you requested in your comments: 
1. Added a short summary of changes introduced in the pull request.
2. Wrote a more descriptive readme including social handle.
3. Added the test.md file to the gitignore file.

They are shown in the top comment and new commits.
I will send another review request.

Thank you!

Alfredbis29 commented on .gitignore/test.md:
```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```
- [ ]  Good job so far but you  still need some changes for your project to look awesome :wink:  , You should ignore the `test.md` and `node modules` before you push on GitHub, They can't be present in your repository, Please  ignore them in `.gitignore`  :grin: :grin:   
This is how it should look like :arrow_down: 

![gitignore](https://user-images.githubusercontent.com/88894525/191379955-23a3aa35-f866-49d3-aa15-6eeb6b7192ce.png)

paulinagonzalezc replied:
Hello @Alfredbis29, I already made the change. Can you let me know if it's ok now? I don't want to fail my last coding review 🥹

paulinagonzalezc replied:
@Alfredbis29 and also can you let me know how you check if they are really being ignored?

Alfredbis29 replied:
hi You can just create a file called .gitignore then you put inside those two files which are test.md and nodes_modules then you push on your branch

paulinagonzalezc replied:
Hi @Alfredbis29 , I did it just as you showed me, can you tell me if I'm correct? 

<img width="1013" alt="Screen Shot 2022-09-20 at 19 49 54" src="https://user-images.githubusercontent.com/90674847/191384480-83e0464b-15e3-4c41-afb4-c74cd7a83180.png">

Alfredbis29 reviewed (changes requested):
Hi @paulinagonzalezc ,

Great work on making the changes requested by a previous reviewer clap You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ![soclose](https://user-images.githubusercontent.com/65233860/190579192-fae86458-021b-42da-bc1d-747b2e7882e3.gif)

## Highlights
* All linters are passing :heavy_check_mark: 
* Implemented correct Github flow  :heavy_check_mark: 
* Good PR description  :heavy_check_mark: 

## Required Changes 
 _Check the comments under the review._

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap:  :clap: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

paulinagonzalezc committed:
Added test.md and node_modules/ to git ignore file

paulinagonzalezc committed:
Added test.md file correctly

paulinagonzalezc committed:
Second try to add test.md to .gitignore

paulinagonzalezc committed:
Another test for test.md file

paulinagonzalezc committed:
Test.md to .gitignore

Reem-lab reviewed (approved):
# Hi @paulinagonzalezc👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯
<img src="https://media1.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>


## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌Professional README ✅

## Optional suggestions
- [OPTIONAL ] Higly recomended,  I think you didn't add the files for stylelint and webhint for testing the linters so kindly add them since they have the rules for organizing your code. and Don't foget to install the packages 🚀✅🙏🏻check this link [here](https://github.com/microverseinc/linters-config/tree/master/html-css)
![image](https://user-images.githubusercontent.com/58553711/191263838-ed6bd75a-c7be-441f-9d13-7cab48a80e19.png)
## If you scroll down you can see how to install their packages then you can run commands to check if there is any error 🙏🏻🚀


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

paulinagonzalezc committed:
Added the files for stylelint

paulinagonzalezc committed:
Added webhint packkages and file

paulinagonzalezc commented:
Hello Reem janina,
I added the files for stylelint and webhint and merged the branch.
But it still appears as review pending in my dashboard. Is there any
problem?


On Tue, Sep 20, 2022 at 8:46 PM Reem janina ***@***.***>
wrote:

> ***@***.**** approved this pull request.
> Hi @paulinagonzalezc <https://github.com/paulinagonzalezc>👋🏻👋🏻
>
> Your project is complete! There is nothing else to say other than... it's
> time to merge it [image: :shipit:]
> Congratulations! 🎉🔥🔥💯💯
>
> <https://camo.githubusercontent.com/025552433f3d232fbb4ff2aa99f4018f1ab4d86321c98dbaf6b3d882641805e5/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f42504a6d74685133595277443651716356442f67697068792e676966>
> Highlights ✨✨
>
> 📌 No Linters error ✅
> 📌 Descriptive pull request✅
> 📌 Great working on the functionality✅
> 📌 Good commit messages ✅
> 📌Professional README ✅
> Optional suggestions
>
>    - [OPTIONAL ] Higly recomended, I think you didn't add the files for
>    stylelint and webhint for testing the linters so kindly add them since they
>    have the rules for organizing your code. and Don't foget to install the
>    packages 🚀✅🙏🏻check this link here
>    <https://github.com/microverseinc/linters-config/tree/master/html-css>
>    [image: image]
>    <https://user-images.githubusercontent.com/58553711/191263838-ed6bd75a-c7be-441f-9d13-7cab48a80e19.png>
>
> If you scroll down you can see how to install their packages then you can
> run commands to check if there is any error 🙏🏻🚀
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/paulinagonzalezc/hello-microverse-projec/pull/1#pullrequestreview-1114615046>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVTZNH5C4EY2KOXYRAEQS7TV7JLFPANCNFSM6AAAAAAQROWVOE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>
